x-pack/plugins/secops/public/pages/home/index.tsx

```diff
@@ -8,9 +8,58 @@ import * as React from 'react';
 import { pure } from 'recompose';
 
 import { ColumnarPage } from '../../components/page';
+import { headers } from '../../components/timeline/body/column_headers/headers';
+import { Sort } from '../../components/timeline/body/sort/sort';
+import { mockDataProviders } from '../../components/timeline/data_providers/mock/mock_data_providers';
+import {
+  OnColumnSorted,
+  OnDataProviderRemoved,
+  OnFilterChange,
+  OnRangeSelected,
+} from '../../components/timeline/events';
+import { Timeline } from '../../components/timeline/timeline';
```

Comment:
I would prefer to name the file index.tsx inside of the folder `timeline` to avoid duplication of a name when we are using it. If possible can we apply to every component inside of folder